src/node_jsvmapi.h

```diff
@@ -188,34 +188,43 @@ NODE_EXTERN napi_status napi_coerce_to_string(napi_env e,
 NODE_EXTERN napi_status napi_get_prototype(napi_env e,
                                            napi_value object,
                                            napi_value* result);
-NODE_EXTERN napi_status napi_property_name(napi_env e,
-                                           const char* utf8name,
-                                           napi_propertyname* result);
 NODE_EXTERN napi_status napi_get_propertynames(napi_env e,
                                                napi_value object,
                                                napi_value* result);
 NODE_EXTERN napi_status napi_set_property(napi_env e,
-                                          napi_value object,
-                                          napi_propertyname name,
+                                          napi_value o,
+                                          napi_value k,
```

Comment:
I'd prefer full words for the parameter names. It may not be entirely obvious that `o`=object, `k`=key, `v`=value. And it should be consistent with the rest of the declarations that mostly spell out parameter names, with the exception of the `e` parameter.